rust/src/frames.rs

```diff
@@ -15,6 +15,8 @@
  * 02110-1301, USA.
  */
 
+//! Module for bindings to the Suricata C frame API.
+
```

Comment:
it's from the `lib.rs` file; line 80.

I don't know what to do with that.

Reply:
Does this happen with all modules rendered after that comment on `lib.rs`? I'll try to check locally.

Reply:
Not at all.

It only happens to frames.rs

Reply:
Thanks for answering :)
I did a simple test, and rustdoc will pick that, if it is as such on the lib.rs file. Maybe it is as such to let the reader know that such file is going to be ignored by cbindgen? If that's the case, then it seems to me that there's nothing to be changed there.

Any input here, @jasonish?

Reply:
Looks like an issue between cbindgen and rustdoc.  But I don't think we need the `cbindgen:ignore` on the frames module, so removing it appear to fixes this.  Please do that, but as its own commit.

Reply:
So, I will remove the `/// cbindgen:ignore` comment totally, right?

Reply:
> So, I will remove the `/// cbindgen:ignore` comment totally, right?

Yes, build to be sure its OK on your end.

Reply:
Sure will.

Reply:
> Looks like an issue between cbindgen and rustdoc.  But I don't think we need the `cbindgen:ignore` on the frames module, so removing it appear to fixes this.  Please do that, but as its own commit.

How do I name the commit? So I don't make a mistake.

Reply:
Hmm, I think it could be:
```
lib: remove cbindgen 'ignore' annotation for frames 

This seemed to cause a conflict between cbindgen and rustdoc.
cbindgen annotation removed since (as discussed) it doesn't seem we 
need that for the frames module.
```

Then we have a good picture of what and why :)

Reply:
OK, but isn't the commit message long? Since the commit standard says not more than 75 characters?

Reply:
> Hmm, I think it could be:
> ```
> lib: remove cbindgen 'ignore' annotation for frames 
> 
> This seemed to cause a conflict between cbindgen and rustdoc.
> cbindgen annotation removed since (as discussed) it doesn't seem we 
> need that for the frames module.
> ```
> 
> Then we have a good picture of what and why :)

OK, but isn't the commit message long? Since the commit standard says not more than 75 characters?

Reply:
For our guidelines, the commit message (first line) has to ideally be within 50 characters, and the commit body (whatever comes after that blank line) should be broken at ~72 characters in length.

We are one 's' longer in the commit message/title (51 char), but i think that's fine. (if someone complains, you're free to blame :P )

Reply:
I'm actually talking about the commit body, it's longer than 75 character length, what to do there?

Reply:
> I'm actually talking about the commit body, it's longer than 75 character length, what to do there?

I see. If you break the lines, as done there, then it is within the 72-character limit, which is fine. 
See a merged example: https://github.com/OISF/suricata/commit/1a132f454a64f699118dafcdfccb0687317b435e

Does this answer your question?

Reply:
> > I'm actually talking about the commit body, it's longer than 75 character length, what to do there?
> 
> I see. If you break the lines, as done there, then it is within the 72-character limit, which is fine. 
> See a merged example: https://github.com/OISF/suricata/commit/1a132f454a64f699118dafcdfccb0687317b435e
> 
> Does this answer your question?

So `This seemed to cause a conflict between cbindgen and rustdoc.` on one line. 

And `cbindgen annotation removed since (as discussed) it doesn't seem we 
need that for the frames module.` on another line, and I'm good to go, right?

Reply:
Maybe something like:
```
rust: remove cbindgen:ignore on frames module

This directive is no longer required, and does mess up the rustdoc description of this module.
```

Wrap the second line as needed. My editor does the wrapping with a keyboard shortcut, others might do it automatically.

Reply:
OK Thank you.

I'll incorporate this as soon as I can and submit a PR.

Reply:
> Thanks for answering :) I did a simple test, and rustdoc will pick that, if it is as such on the lib.rs file. Maybe it is as such to let the reader know that such file is going to be ignored by cbindgen? If that's the case, then it seems to me that there's nothing to be changed there.
> 
> Any input here, @jasonish?

I don't think the `cbindgen:ignore` should show up in the output like that.

Reply:
Is that something with rustdoc settings, or a bug? or nothing?